Angular/src/app/paradigm-edit/paradigm-edit.component.html

```diff
@@ -8,8 +11,13 @@ <h1>
 	<div class="root-column"> 
 		<h2> Root Words </h2> 
 		<ul>
-		  <li *ngFor="let root of roots$; let index = index" (click)="onSelect(index)">
-		    <a>{{ root }}</a>
+		  <li *ngFor="let root of roots$; let index = index" class="root-item-container">
+		    <div (click)="onSelect(index)" class="root-item">
+		    	<a>{{ root }}</a> 
+		    </div> 
+		    <div class="root-delete">
+		    	<a (click)="deleteWord(index)"><i class="material-icons">delete</i></a>
```

Comment:
This is nitpicky, and I'll leave the implementation up to you, but should the (click)="function()" references be in the same tag types? Either both inside of the < a > tag or both inside of the < div > tag?

Reply:
Good call here as well, I think it's better to stay consistent with this. I updated it in the most recent commit